app/assets/stylesheets/navbar.scss

```diff
@@ -9,22 +9,27 @@
 
 .navbar-nav li a {
   color: white;
-  font-family: 'Open Sans', sans-serif;
+  font-family: 'Questrial', sans-serif;
   font-size: 20px;
+  font-weight: bold;
 }
 
 .navbar-nav li a:hover {
-  background-color: #169cb7;
+  background-color: #48a610;
 }
 
 .navbar-nav li a:focus {
-    background-color: #grey;
+    background-color: #48a610;
```

Comment:
Not something you need to do now, but for future reference: Look up setting up css variables/constants. Since you're using the same colour a lot, having a variable called something like e.g. 'LS-green' and using that to define all these background-colors, means that if you ever decide to change this colour, then you only need to change it in one spot.